schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/balance/BalanceService.java

```diff
@@ -36,21 +37,21 @@
      *
      * @param payment - a payment that has been executed and needs to be reflected on the balance
      */
-    void payment(Payment payment);
+    void payment(Payment payment) throws ModelException;
 
-    void unpaidSubscription(Key subscriptionId) throws EntityNotFoundException;
+    void unpaidSubscription(Key subscriptionId) throws ModelException;
```

Comment:
Doesn't throw `ModelException`...  Should only throw `EntityNotFoundException`.  Same applies for all the others